front/src/modules/pipeline-processes/components/Board.tsx

```diff
@@ -70,11 +71,14 @@ export function Board({
 }: BoardProps) {
   const [board, setBoard] = useRecoilState(boardColumnsState);
   const [items, setItems] = useRecoilState(boardItemsState);
+  const [selectedItemKeys, setSelectedItemKeys] = useRecoilState(
+    selectedBoardItemsState,
```

Comment:
So I should have been more precise, the convention is this : 

```
const [selectedItemKeys, setSelectedItemKeys] = useRecoilState(selectedItemKeysState)

// OR 

const [selectedBoardItems, setSelectedBoardItemsState] = useRecoilState(selectedBoardItemsState)
```

Since selectedItems is little vague, it should be better to include board in the naming